### ISSUE TITLE
"I..."

### ISSUE THREAD

**Issue by SkyTheLeafeon** [andwn/cave-story-md#289]
Curly says this if the player checks her after the Core fight.
This should not happen.

**Comment by Leo40Git** [comment]
IIRC she's not even supposed to be intractable in that scenario. Also, this probably means she's intractable in the Core fight itself, which would kinda break the flow if you accidentally pressed down.

**Comment by SkyTheLeafeon** [comment]
Hmm... I'll have to check that the next time I do a run of the MD version.

**Comment by andwn** [comment]
AI Curly is a different object than NPC Curly, so you won't be able to interact while fighting the Core. 

I found this...

![code_2018-04-11_22-04-47](https://user-images.githubusercontent.com/8164394/38652543-6c1df93a-3dd4-11e8-8f62-310d8f4a3e54.png)

NXEngine doesn't have the problem, but does the same (that's where I got it from after all):

![firefox_2018-04-11_22-32-36](https://user-images.githubusercontent.com/8164394/38653343-4768ff14-3dd8-11e8-86f5-53f21ba5f0ba.png)

So actually, it turned out to be a stupid nuance in my engine. `eflags` come from the PXE. `nflags` come from the NPC table. I can't give any honest answer as to why this fixed it. This is what I get for not combining the flag variables I guess...

![code_2018-04-11_22-24-47](https://user-images.githubusercontent.com/8164394/38653127-2b3a9e8e-3dd7-11e8-9868-cd12de3a0421.png)

![blastem_2018-04-11_22-21-15](https://user-images.githubusercontent.com/8164394/38653165-6be2752e-3dd7-11e8-9abf-c71e68b0a3a3.png)
